migrate.go

```diff
@@ -520,11 +544,11 @@ func (ms MigrationSet) ExecMax(db *sql.DB, dialect string, m MigrationSource, di
 }
 
 // Plan a migration.
-func PlanMigration(db *sql.DB, dialect string, m MigrationSource, dir MigrationDirection, max int) ([]*PlannedMigration, *gorp.DbMap, error) {
-	return migSet.PlanMigration(db, dialect, m, dir, max)
+func PlanMigration(db *sql.DB, dialect string, m MigrationSource, dir MigrationDirection, max int, version int64) ([]*PlannedMigration, *gorp.DbMap, error) {
```

Comment:
Sorry to be difficult again, but we cannot have changes that break compatibility with the public API.

I'd leave this version unchanged and add a `PlanMigrationToVersion(db *sql.DB, dialect string, m MigrationSource, dir MigrationDirection, version int64)`

Reply:
Understood. Do I also need to keep the PlanMigration method for MigrationSet?
```
func (ms MigrationSet) PlanMigration(db *sql.DB, dialect string, m MigrationSource, dir MigrationDirection, max int) ([]*PlannedMigration, *gorp.DbMap, error) 
```

Reply:
Updated. Now the PR should be good for all the existing public APIs.